src/lib/Client.ts

```diff
@@ -408,6 +421,11 @@ export type NetworkOptions = {
   appVersion?: string
 }
 
+export type KeyType = {
```

Comment:
Should move this to types folder

Reply:
Actually there's a few other types defined there, will hold off on pattern change for now